src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -116,7 +119,54 @@ async function boot(options?: Partial<WebAssemblyStartOptions>): Promise<void> {
     }
   };
 
+  type BeforeBlazorStartedCallback = (options: Partial<WebAssemblyStartOptions> | undefined, bootConfigResul: BootConfigResult) => Promise<void>;
+  type AfterBlazorStartedCallback = () => Promise<void>;
+  type BlazorInitializer = { beforeBlazorStarts: BeforeBlazorStartedCallback, afterBlazorStarted: AfterBlazorStartedCallback };
```

Comment:
In terms of naming, do you think it would be reasonable to change to something like the following?

```suggestion
  type BlazorInitializer = { beforeStart: BeforeBlazorStartedCallback, afterStart: AfterBlazorStartedCallback };
```

Reply:
I've done this change, however I'm a bit worried about using such common names. If you don't share that concern, let's keep it this way.

Reply:
(tp be clear, I mean with your changes)